nipype/interfaces/dcm2nii.py

```diff
@@ -304,7 +304,7 @@ def _parse_stdout(self, stdout):
                         bvals.append(out_file + ".bval")
                         find_b = False
                 # next scan will have bvals/bvecs
-                elif 'DTI gradients' in line or 'DTI gradient directions' in line:
+                elif 'DTI gradients' in line or 'DTI gradient directions' in line or 'DTI vectors' in line:
```

Comment:
@leej3 - i'm going to merge this in, but it would be good to submit an issue to dcm2niix to make this string uniform across scanners/dicoms.